src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -1239,17 +1340,26 @@ print(animal.name); // "Bob"
 print(animal.age); // 5
 ```
 
+To make the first cascade syntax null-aware, write it as `?..`.
+
+{:.include-lang}
+```dart
+var result = maybePerson
+    ?..employment = employer
+    ..salary = salary;
+```
+
+Dart ignores the entire cascade if the `maybePerson` value is `null`.
+
 ## Collections
 
-This section covers some collection types
-in Dart and how they compare to their equivalents
-in JavaScript.
+This section covers some collection types in Dart and compare them
+to similar types in JavaScript.
 
 ### Lists
 
-List literals are defined the same way in Dart as
-arrays are defined in JavaScript,
-using square brackets and separated by commas: 
+Dart writes list literals as JavaScript arrays.
```

Comment:
This text is a bit confusing to me. It seems to imply Dart uses JavaScript arrays somehow.

Perhaps:

```suggestion
Dart writes list literals similarly to JavaScript arrays.
```